src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportHistory/ErrorReportHistoryViewController.swift

```diff
@@ -0,0 +1,54 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+class ErrorReportHistoryViewController: DynamicTableViewController, DismissHandling {
+	
+	// MARK: - Init
+
+	init() {
+		self.viewModel = ErrorReportHistoryViewModel()
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Overrides
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setupTableView()
+	}
+	
+	// MARK: - Private
+	
+	private let viewModel: ErrorReportHistoryViewModel
+
+	private func setupTableView() {
+		navigationItem.largeTitleDisplayMode = .never
+		view.backgroundColor = .enaColor(for: .background)
+		tableView.separatorStyle = .none
+		
+		tableView.register(
+			UINib(nibName: String(describing: ErrorReportHistoryCell.self), bundle: nil),
+			forCellReuseIdentifier: CustomCellReuseIdentifiers.historyCell.rawValue
+		)
+
+		dynamicTableViewModel = viewModel.dynamicTableViewModel
+	}
+
+}
+
+// MARK: - Cell reuse identifiers.
+
+ extension ErrorReportHistoryViewController {
```

Comment:
this can be fileprivate ?

Reply:
I am using this for referencing it in view controller i.e., ErrorReportHistoryViewController.CustomCellReuseIdentifiers.historyCell, don't like hardcoded strings

Reply:
The whole `ReuseIdentifiers`-thing is a mess. Imho, this belongs into the UITableViewCell. Now you have a strong dependency between cell and view controller.

(No need to act on this comment, this is more a refactoring candidate)

Reply:
Normally, we should store all the constants for reuseidentifiers like we have for Appstrings and should not have strings lying around in the entire application.